cle/backends/elf/metaelf.py

```diff
@@ -11,6 +12,7 @@
 
 __all__ = ('MetaELF',)
 
+l = logging.getLogger('cle.backends.elf.metaelf')
```

Comment:
`logging.getLogger(name=__file__)`

Reply:
The string logger name I used here is following the pattern of the rest of the code. Should I change it to use `__file__` instead?

Reply:
Umm... angr has switched to `name=__file__` a long time ago. Maybe it's time for CLE to switch, too.

Reply:
I'll change it here, and file a larger PR to overhaul the logger names too

Reply:
I have updated this line to match the new convention in angr's repo. I've also [filed an issue](https://github.com/angr/cle/issues/219) to update the remainder of CLE logger names.